diff --git a/config.sub b/config.sub
index 9633db7..c6647ff 100755
--- a/config.sub
+++ b/config.sub
@@ -1506,6 +1506,8 @@ case $os in
 		;;
 	-nacl*)
 		;;
+	-HaLVM)
+		;;
 	-none)
 		;;
 	*)
diff --git a/mkGmpDerivedConstants/ghc.mk b/mkGmpDerivedConstants/ghc.mk
index 6c06224..6c82fed 100644
--- a/mkGmpDerivedConstants/ghc.mk
+++ b/mkGmpDerivedConstants/ghc.mk
@@ -17,7 +17,7 @@ libraries/integer-gmp/mkGmpDerivedConstants_dist_INSTALL         = YES
 libraries/integer-gmp/mkGmpDerivedConstants_dist_INSTALL_INPLACE = YES
 libraries/integer-gmp/mkGmpDerivedConstants_dist_EXTRA_CC_OPTS += $(gmp_CC_OPTS)
 
-$(eval $(call build-prog,libraries/integer-gmp/mkGmpDerivedConstants,dist,1))
+$(eval $(call build-prog,libraries/integer-gmp/mkGmpDerivedConstants,dist,0))
 
 GmpDerivedConstants_HEADER = libraries/integer-gmp/mkGmpDerivedConstants/dist/GmpDerivedConstants.h
 
